/lidar_2/lslidar_driver_node:
  ros__parameters: 
    frame_id: laser_link                                    #激光坐标
    group_ip: 224.1.1.2
    add_multicast: false 
    device_ip: 192.168.1.222                                #雷达源ip
    device_ip_difop: 192.168.1.125                          #雷达目的IP
    msop_port: 2368                                         #雷达目的端口号
    difop_port: 2369                                        #雷达源端口号
    agreement_type: 1.6                                     #雷达协议选择:1-1.7协议   2-1.6协议 
    angle_disable_min: 0.0                                  #角度裁剪开始值
    angle_disable_max: 0.0                                  #角度裁剪结束值
    min_range: 0.0                                          #雷达接收距离最小值
    max_range: 200.0                                        #雷达接收距离最大值
    use_gps_ts: false                                       #雷达是否使用GPS授时
    scan_topic: /scan                                       #设置激光数据topic名称
    pubScan: true                                           #是否发布scan话题
    pubPointCloud2: true                                    #是否发布pointcloud2话题
    pointcloud_topic: /lslidar_point_cloud                  #设置激光数据topic名称
#    pcap: /home/ls/work/2211/M10_P_gps.pcap                 #雷达是否使用pcap包读取功能
